src/components/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+export function Header(): JSX.Element {
+  return (
+    <header className="p-3 mb-3 align-right" style={{ background: '#009cab' }}>
+      <div className="container">
+        <HeaderLink to="/">
```

Comment:
you probably don't _style_ the logo differently when you're on the home page. Perhaps use a regular `Link`?

Reply:
I've just used regular `Link`'s for the other header links too :)